### PR TITLE
Planner support

### PR DESCRIPTION
**Pull request by asaba96**
Added planner support and launching. Additionally, added parameters for arena min/max sizes. Finally, added test planner abstract.

### CODE REVIEW

File: launch/test_planner.launch [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<launch>
+    <arg name="platform" default="sim" doc="string representing the platform" />
+    <node name="test_planner_abstract" pkg="iarc7_abstract" type="test_planner.py" />
+    <param name="safety_bonds/test_planner_abstract/form_bond" value="true" />
+
+    <include file="$(find iarc7_launch)/launch/main.launch">
```
Comment (amiller27) [inline launch/test_planner.launch]: I don't think we want `main` here, we probably want `passive_stack` with some other stuff

Comment (asaba96) [reply launch/test_planner.launch]: Main launches `passive stack` and the motion stack.

Comment (amiller27) [reply launch/test_planner.launch]: But it won't in the future, that's just what it does now because that's what we have implemented.

Comment (amiller27) [reply launch/test_planner.launch]: In the future it'll launch a bunch of AI stuff and possibly other things you don't need.

Comment (asaba96) [reply launch/test_planner.launch]: Okay will change

Comment (asaba96) [reply launch/test_planner.launch]: fixed

**Review by amiller27 — APPROVED** [review]

**Pull request merged**